tests/fixtures/ecma-version/9/regexp-lookbehind-assertion/valid-negative-1.result.js

```diff
@@ -47,7 +47,7 @@ module.exports = {
                     0,
                     8
                 ],
-                "value": null,
+                "value": process.versions.node >= "9.0.0" ? new RegExp(String.raw`(?<!a)`) : null,
```

Comment:
emm, this failed. seems Node.js 8 doesn't support these features, checking Node.js version seems not a good idea, thougths?

Reply:
For unicode regexes, the tests use the [`conditionalRegex`](https://github.com/eslint/espree/blob/0252144395e4a56ddaacb1a0b8eba693391e3d62/tests/lib/conditional-regex-value.js) helper. Would that work here?

It's not related to the issue, but I think using `>= "9.0.0"` will fail when Node 10.0.0 comes out, because it compares the version alphabetically.